### ISSUE TITLE
Does Softmax loss output layer normalise the loss?

### ISSUE THREAD

**Issue by horserma** [dmlc/mxnet#1517]
In Caffe, the loss layer can normalise the loss by dividing the label number. I am wondering if the loss layer in Mxnet normalises the result. I tried @tornadomeet FCN, and I found when I used 1e-9, it would be so small to get weights updated. I am a little confused here.

**Comment by piiswrong** [comment]
It does normalize the loss. So use a larger learning rate

**Comment by tornadomeet** [comment]
@horserma it's due to the changed code in [softmax_output-inl.h](https://github.com/dmlc/mxnet/commit/ea259985d839f0a77c03703ab7271440bc9ddc2f) after the fcn-xs example, so you can use larger lr for the new version of mxnet.
